vllm/attention/backends/rocm_flash_attn.py

```diff
@@ -511,6 +515,11 @@ def __init__(
                     self.use_naive_attn = True
 
             if self.use_naive_attn:
+                if logits_soft_cap is not None:
```

Comment:
You make sure it's not None in the constructor. So naive flash attention can never be used now

Reply:
fixed. thanks.